modules/gdscript/editor/script_templates/EditorScenePostImport/basic_import_script.gd

```diff
@@ -0,0 +1,11 @@
+# meta-description: Basic import script template
+@tool
+extends EditorScenePostImport
+
+
+# Called by the editor when a scene has this script set as the import script in the import tab.
+func _post_import(scene: Node) -> Object:
+	# Modify the contents of the scene upon import. For example, setting up LODs:
+#	(scene.get_node(^"HighPolyMesh") as MeshInstance3D).draw_distance_end = 5.0
+#	(scene.get_node(^"LowPolyMesh") as MeshInstance3D).draw_distance_begin = 5.0
```

Comment:
Add missing indentation:

```suggestion
	#	(scene.get_node(^"HighPolyMesh") as MeshInstance3D).draw_distance_end = 5.0
	#	(scene.get_node(^"LowPolyMesh") as MeshInstance3D).draw_distance_begin = 5.0
```

Reply:
There is indentation, it's after the #. When commenting out code using the editor shortcut (Ctrl + K) it does it this way, so I formatted it the same way. Should I not?

Reply:
In that case, I guess it makes sense to keep things as they are.